index.js

```diff
@@ -61,15 +60,14 @@ module.exports = {
         }.bind(this));
       },
       _logSuccess: function(outputPath) {
-        var self = this;
```

Comment:
within this method, `self` was used over `bind(this)`. I've changed functions to `bind` to unify code with other methods.